webpack/scenes/Subscriptions/Manifest/ManifestHistoryTableSchema.js

```diff
@@ -6,6 +6,9 @@ export const columns = [
     header: {
       label: __('Status'),
       formatters: [headerFormat],
+      props: {
+        index: 0,
```

Comment:
What are the indexes good for?

Reply:
They're needed after the changes to the table in https://github.com/Katello/katello/pull/7331.  Basically to support selecting rows.